src/android/InAppBrowser.java

```diff
@@ -132,8 +133,10 @@ Licensed to the Apache Software Foundation (ASF) under one
     private boolean mediaPlaybackRequiresUserGesture = false;
     private boolean shouldPauseInAppBrowser = false;
     private boolean useWideViewPort = true;
-    private ValueCallback<Uri[]> mUploadCallback;
+    private ValueCallback<Uri> mUploadCallback;
+    private ValueCallback<Uri[]> mUploadCallbackLollipop;
```

Comment:
Removing code that handles API < 21 should also eliminate the need to have two different class members to handle different API levels.